sdk/containerregistry/Microsoft.Azure.Management.ContainerRegistry/tests/Tests/ContainerRegistryTests.cs

```diff
@@ -340,6 +342,106 @@ public void ContainerRegistryReplicationTest()
             }
         }
 
+        [Fact]
+        public void ContainerRegistryTransferExportTest()
+        {
+            var handler = new RecordedDelegatingHandler { StatusCodeToReturn = HttpStatusCode.OK };
+
+            using (MockContext context = MockContext.Start(this.GetType()))
+            {
+                var resourceClient = ContainerRegistryTestUtilities.GetResourceManagementClient(context, handler);
+                var registryClient = ContainerRegistryTestUtilities.GetContainerRegistryManagementClient(context, handler);
+
+                // Create resource group
+                var resourceGroup = ContainerRegistryTestUtilities.CreateResourceGroup(resourceClient);
+
+                // Create container registry and exportPipeline
+                var registry = ContainerRegistryTestUtilities.CreateManagedContainerRegistry(registryClient, resourceGroup.Name, resourceGroup.Location);
+                var exportPipeline = ContainerRegistryTestUtilities.CreatedContainerRegistryExportPipeline(registryClient, resourceGroup.Name, registry.Name, resourceGroup.Location);
```

Comment:
>CreatedContainerRegistryExportPipeline [](start = 68, length = 38)

While abstract the creation part promote reuse, I feel in this case, it's better to put the creation code in test file directly. 

1. The test is to test the pipeline creation code, if I read the code I want to know a little more how it is done.

2. If we share the code link as sample, it will be easy read it from one file.

Reply:
Moved the create code to the test class.